src/test_Locations.py

```diff
@@ -0,0 +1,6 @@
+from Locations import Locations
+
+def test_locations():
+
+    location = Locations('https://www.rrc.texas.gov/resource-center/research/data-sets-available-for-download/')
```

Comment:
need to expand this obviously
but what it needs to do is open a location and then scrape that location appropriately b/c TX RRC links to datasets does not follow a naming convention but a file structure that includes a genereated hash, e.g.

the Oil Ledger list of current ebc files for Districts 1,2,3 are under:

https://mft.rrc.texas.gov/link/c5081c77-d32c-4ded-9b33-5aca3833306c